crates/fj-kernel/src/algorithms/sweep/mod.rs

```diff
@@ -47,4 +47,6 @@ pub trait Sweep: Sized {
 pub struct SweepCache {
     /// Cache for global vertices
     pub global_vertex: BTreeMap<ObjectId, Handle<Vertex>>,
+    /// Cache for global edges
+    pub global_edge: BTreeMap<ObjectId, Handle<GlobalEdge>>,
```

Comment:
Maybe we should have a more general caching solution. This seems to work for now but IDK if it's ideal.

Reply:
I don't know, this seems fine to me. If you come up with something better, feel free to suggest it of course, but I think we can keep it as-is.